### PR TITLE
Don't add platform flags when they are already specified.

### PR DESCRIPTION
**Pull request by meteorcloudy**
To fix https://github.com/bazelbuild/bazel-toolchains/pull/237

@xingao267 Does this change look good to you?
Will this fix https://github.com/bazelbuild/continuous-integration/issues/376?

### CODE REVIEW

File: buildkite/bazelci.py [hunk 1]
```diff
@@ -861,20 +861,36 @@ def rbe_flags(accept_cached):
...
         "--action_env=BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN=1",
-        # Platform flags:
-        # The toolchain container used for execution is defined in the target indicated
-        # by "extra_execution_platforms", "host_platform" and "platforms".
-        # If you are using your own toolchain container, you need to create a platform
-        # target with "constraint_values" that allow for the toolchain specified with
-        # "extra_toolchains" to be selected (given constraints defined in
-        # "exec_compatible_with").
-        # More about platforms: https://docs.bazel.build/versions/master/platforms.html
         "--extra_toolchains=@bazel_toolchains//configs/ubuntu16_04_clang/latest:toolchain_default",
```
Comment (xingao267) [inline buildkite/bazelci.py]: Should these be deleted?

File: buildkite/bazelci.py [hunk 2]
```diff
@@ -824,7 +824,7 @@ def common_flags(bep_file, platform):
             "--experimental_multi_threaded_digest"]
 
 
-def rbe_flags(accept_cached):
+def rbe_flags(original_flags, accept_cached):
```
Comment (xingao267) [inline buildkite/bazelci.py]: the original_flags contain the user specified flags in the yml file I assume?

Comment (meteorcloudy) [reply buildkite/bazelci.py]: Yes.

File: buildkite/bazelci.py (hunk 1 above)
Comment (meteorcloudy) [reply buildkite/bazelci.py]: There are moved outside.

**Comment by meteorcloudy** [comment]
I'll merge this to see if bazel-toolchain is fixed.

File: buildkite/bazelci.py (hunk 1 above)
Comment (meteorcloudy) [reply buildkite/bazelci.py]: Oh, my mistake, they should be deleted.

Comment (meteorcloudy) [reply buildkite/bazelci.py]: Done in https://github.com/bazelbuild/continuous-integration/commit/298e6155a80913d0c81993228ea38b4151ee1bd0

**Comment by meteorcloudy** [comment]
It's finally green!
https://buildkite.com/bazel/bazel-with-downstream-projects-bazel/builds/592#538e9984-fd3e-459c-9d75-e7c5e5c35a1f

**Comment by xingao267** [comment]
Yeah!! Thanks for making those changes in CI!

**Pull request merged**